src/robusta/core/reporting/blocks.py

```diff
@@ -467,13 +473,28 @@ class PrometheusBlock(BaseBlock):
     data: PrometheusQueryResult
     metadata: Dict[str, str]
 
-    def __init__(self, data: PrometheusQueryResult, query: str):
+    vertical_lines: Optional[List[PrometheusBlockLineData]]
+    horizontal_lines: Optional[List[PrometheusBlockLineData]]
+    y_axis_type: Optional[ChartValuesFormat]
+    graph_name: Optional[str]
+
+    def __init__(self, data: PrometheusQueryResult, query: str, vertical_lines: Optional[List[PrometheusBlockLineData]],
```

Comment:
This should have default None to all optional fields, because this way it breaks all other usages of PrometheusBlock
```
def __init__(self, data: PrometheusQueryResult, query: str, 
vertical_lines: Optional[List[PrometheusBlockLineData]] = None,
                 horizontal_lines: Optional[List[PrometheusBlockLineData]] = None,
                 y_axis_type: Optional[ChartValuesFormat] = None,
                 graph_name: Optional[str] = None):
```